src/main/java/uk/gov/companieshouse/model/Attachment.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.companieshouse.model;
+
+import java.util.Objects;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class Attachment {
+
+    @NotBlank(message = "attachement id must not be null.")
+    private String id;
+
+    @NotBlank(message = "name must not be blank.")
+    private String name;
+
+    @NotBlank(message = "contentType must not be blank.")
+    private String contentType;
+
+    @NotNull()
+    private Integer size;
+
+    public String getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getContentType() {
+        return contentType;
+    }
+
+    public int getSize() {
+        return size;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setContentType(String contentType) {
+        this.contentType = contentType;
+    }
+
+    public void setSize(int size) {
```

Comment:
this setter for the `size` field accepts the primitive int and not the wrapper Integer, please update this